scripts/dex-deploy.ts

```diff
@@ -0,0 +1,29 @@
+import { ethers, upgrades } from 'hardhat';
+
+async function main() {
+  // ethers is avaialble in the global scope
+  const [deployer] = await ethers.getSigners();
+  console.log(
+    'Deploying the contracts with the account:',
+    await deployer.getAddress()
+  );
+
+  console.log('Account balance:', (await deployer.getBalance()).toString());
+
+  const Contract = await ethers.getContractFactory('DEX');
+  const contract = await upgrades.deployProxy(
```

Comment:
it's a very bad practice to have two variables that differ only in their case.

Reply:
agreed, changed